src/nxt_conf_validation.c

```diff
@@ -2904,7 +2908,7 @@ nxt_conf_vldt_object_iterator(nxt_conf_validation_t *vldt,
 {
     uint32_t                index;
     nxt_int_t               ret;
-    nxt_str_t               name;
+    nxt_str_t               name, str;
```

Comment:
Ah, this hunk should be dropped ;)

str is not used...

Reply:
Hmmm, true!